client/component/people.js

```diff
@@ -4,30 +4,29 @@ import { IconCircleCheck } from "./icon";
 import { UtilityText } from "./text";
 import { HoverView, Pad, PadBox, spacings } from "./basics";
 import { formatDate, formatMiniDate } from "./date";
-import { colorBlack, colorDisabledBackground, colorTextGrey, colorWhite, colorPink } from "./color";
+import { colorBlack, colorDisabledBackground, colorTextGrey, colorWhite, colorPink, colorGreyHover, colorGreyFormHover } from "./color";
 import { TextButton } from "./button";
 import { useLanguage } from "./translation";
 import { useConfig } from "../util/features";
 
-export function ProfilePhoto({ userId, type = 'large', photo = null, faint = false, check = false, border = false }) {
-
+export function ProfilePhoto({ userId, type = 'large', photo = null, faint = false, border = false }) {
     const persona = useObject('persona', userId);
     const isLive = useIsLive();
     const meKey = usePersonaKey();
     const { profilePhotoLayers } = useConfig();
 
     if (meKey == userId && isLive) {
-        return <MyProfilePhoto type={type} photo={photo} faint={faint} check={check} border={border} />
+        return <MyProfilePhoto type={type} photo={photo} faint={faint} border={border} />
     } else {
         const face = persona?.face;
         if (face || photo || persona?.photoUrl) {
             if (profilePhotoLayers?.length > 0) {                
                 return <> 
-                    <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} border={border} faint={faint} check={check} />
+                    <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} border={border} faint={faint} />
                     {renderLayers(type, persona, profilePhotoLayers)}
                 </>
             } else {
-                return <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} border={border} faint={faint} check={check} />
+                return <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} border={border} faint={faint} />
```

Comment:
This reminds me that we probably need to show profilePhotoLayers for LetterFace and AnonymousFace (which I don't think we are actually using right now) too - since a person with a celebrity name may not be showing a profile photo.

If we do that, then we probably want to turn the photo layers into a component that can easily wrap any photo, so we can easily re-use the same code in all these locations - and also in the login screen.

Reply:
done